scripts/exrrfs_ioda_bufr.sh

```diff
@@ -259,25 +259,33 @@ cp "${RDASAPP_DIR}"/rrfs-test/IODA/offline_vad_thinning.py .
 # offline domain check & patch
 for ioda_file in ioda*nc; do
   grid_file="${FIX_GSI}/${PREDEF_GRID_NAME}/fv3_grid_spec"
-  if [[ "${ioda_file}" == *abi* ]]; then
+  if [[ "${ioda_file}" == *abi* && "${ioda_file}" != *satwnd* ]]; then
```

Comment:
Thanks for this! I actually found this today as well and was going to update after my current PR (currently a draft) gets in!